pkg/types/int.go

```diff
@@ -58,11 +60,16 @@ func (i *Int) UnmarshalJSON(data []byte) error {
 	return nil
 }
 
+func (i Int) String() string {
+	return fmt.Sprintf("%d", i.Int64)
+}
+
```

Comment:
Something is missing here due to i is not just an int. Also while you're on it consider FormatUint.

Reply:
And I don't mean the (also) missing doc.

Reply:
> Something is missing here due to i is not just an int

Of course it's just an `Int`. I don't know what this comment is supposed to tell me.